.github/workflows/install-script-pwsh-test.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Meta comment: I don't expect that we update the PowerShell script nor the microsoftgoinstallscript package often, maybe one or two times per year, if any. On the other hand, the go-infra is updated quite more often. Also, microsoftgoinstallscript doesn't depend on other go-infra packages.

With this in mind, I would suggest planting a go.mod file in install/powershellscript so that package gets promoted to being it's own module. Then we can version it independently of 'go-infra'. Well, we are currently versioning go-infra, but I think we should definitely version microsoftgoinstallscript.

Reply:
Huh, I've always assumed those still get versioned with the surrounding module, but finally saw https://go.dev/doc/modules/managing-source#multiple-module-source for how that works.

Yeah, we would produce a lot of useless dependabot update PRs for people if we didn't version it separately.